webassembly/README.md

```diff
@@ -0,0 +1,5 @@
+WebAssembly sample
+===
+See https://googlechrome.github.io/samples/webassembly/index.html for a live demo.
+
+Learn more at https://developers.google.com/web/updates/emscripting-a-c-library
```

Comment:
If you think* it will happen in march, I think it's safe to set the appropriate URL: https://developers.google.com/web/updates/2018/03/emscripting-a-c-library

Reply:
Good call